package.json

```diff
@@ -48,6 +48,6 @@
   },
   "scripts": {
     "start": "./start.sh",
-    "load": "curl -O $HUBOT_ENV_LOCATION && source ./.env && ./bin/hubot --adapter mattermost --name $MATTERMOST_HUBOT_USERNAME --alias CFPBot"
+    "load": "curl -O $HUBOT_ENV_LOCATION && source ./.env && touch ./.env_private && source ./.env_private && ./bin/hubot --adapter mattermost --name $MATTERMOST_HUBOT_USERNAME --alias CFPBot"
```

Comment:
great question. the touch just ensures that the file exists so that sourcing it doesn't fail.

The purpose is to allow us to keep env vars directly on the server, not stored in SCM anywhere. this means that on the server, for now, .env_private is manually updated.

This should be a temporary solution. Right now, the `$HUBOT_ENV_LOCATION` resolves to an internal github repo that is not private. And the stuff I need to store in `.env_private` I am not putting in a non-private repo. Once we sort that out, I'll consider keeping those things in github